openslides/users/static/js/users/site.js

```diff
@@ -316,20 +326,9 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                     type: 'input',
                     templateOptions: {
                         label: gettextCatalog.getString('Default password'),
```

Comment:
What about renaming this field (now) to `First password` or `Initial password` or something like this?